Python/sysmodule.c

```diff
@@ -838,20 +887,20 @@ static PyStructSequence_Desc asyncgen_hooks_desc = {
     2
 };
 
+/*[clinic input]
+sys.set_asyncgen_hooks
+
+    firstiter: object = NULL
```

Comment:
The signature will contain `None` as a default value for both arguments. But passing `None` and bot passing an argument have different effects. It is not possible to use Argument Clinic for this function.

There is also a bug in the signature of this function in the current code. It should be `set_asyncgen_hooks(* [, firstiter] [, finalizer])`, not `set_asyncgen_hooks(*, firstiter=None, finalizer=None)`.